daos/stored_recipes/test/stored_recipes_logic_test.py

```diff
@@ -0,0 +1,49 @@
+import unittest
+
+from flask import Flask
+import recipes_mocks
+from daos.database import DataBase
+from daos.stored_recipes import stored_recipes_manager
```

Comment:
verify deletion in the test, and take advantage of it to make sure we are not creating garbage while testing. the database should be empty.